### PR TITLE
Migrate SKY UX 3

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -28,4 +28,4 @@ before_install:
 
 # Identical to package.json except for travis wait
 script:
-  - skyux lint && skyux test && skyux e2e && travis_wait skyux build --logFormat none --output-keep-alive
+  - skyux lint && skyux test && skyux e2e && skyux build
```
Comment (Blackbaud-BobbyEarl) [inline .travis.yml]: We had to introduce the `travis_wait` due to long build times.  Is there a reason to remove it now?

Comment (Blackbaud-SteveBrush) [reply .travis.yml]: Ah, this was unintentional (though, removing it doesn't seem to affect the build). I'll put it back.

Comment (Blackbaud-BobbyEarl) [reply .travis.yml]: Thanks!

**Comment by Blackbaud-BobbyEarl** [comment]
The content in the Code sections on the component pages seems to be missing.  Each one produces 'No demo files found for: [ComponentName]`.

**Comment by blackbaud-johnly** [comment]
@Blackbaud-BobbyEarl, which components are missing code sections? I caught that the repeater section was missing the code sample, and Steve fixed that yesterday. I didn't notice any others yesterday, and I don't see any now that I'm clicking back through to double-check.

**Comment by Blackbaud-BobbyEarl** [comment]
Clarifying my comment, I see no code samples on the PR preview.  Serving locally, they work.

https://developer.blackbaud.com/skyux?_sp.spa=pr-migrate-skyux3

**Comment by blackbaud-johnly** [comment]
I can confirm that I'm seeing the same thing as Bobby. When serving locally, all the components include code in the Cod sections. But I don't see code in any of the components in the PR review.

**Pull request merged**